File: BankingTool.Model/Dto/RequestDto/BankAccount/GetToAccountListExcludedByAccountIdRequest.cs
using BankingTool.Model.Dto.RequestDtos;

namespace BankingTool.Model.Dto.RequestDto.BankAccount
{
    public class GetToAccountListExcludedByAccountIdRequestObject
    {
        public GetToAccountListExcludedByAccountIdRequest Request { get; set; }
    }
    public class GetToAccountListExcludedByAccountIdRequest
    {
        public GetToAccountListExcludedByAccountIdRequestAccount Account { get; set; }
    }
    public class GetToAccountListExcludedByAccountIdRequestAccount : RequestId
    {
    }
}

File: bankingtool.client/src/app/dto/request/bank-account/get-to-account-list-excluded-by-account-id-request.ts
import { RequestId } from "../request-id";

export interface GetToAccountListExcludedByAccountIdRequestObject {
  request: GetToAccountListExcludedByAccountIdRequest;
}

export interface GetToAccountListExcludedByAccountIdRequest {
  account: GetToAccountListExcludedByAccountIdRequestAccount;
}

export interface GetToAccountListExcludedByAccountIdRequestAccount extends RequestId {

}

export const CreateGetToAccountListExcludedByAccountIdRequestObject = (accountId: number): GetToAccountListExcludedByAccountIdRequestObject => {
  let model: GetToAccountListExcludedByAccountIdRequestObject = {
    request: {
      account: {
        id: accountId,
      }
    }
  }
  return model;
}
